interface/wx/clrpicker.h

```diff
@@ -44,7 +44,20 @@ wxEventType wxEVT_COLOURPICKER_CHANGED;
            The user changed the colour selected in the control either using the
            button or using text control (see @c wxCLRP_USE_TEXTCTRL; note that
            in this case the event is fired only if the user’s input is valid,
-           i.e. recognizable).
+           i.e. recognizable). When using a popup dialog for changing the
+           colour, this event is sent only when the changes in the dialog are
+           accepted by the user, unlike @c EVT_COLOURPICKER_CURRENT_CHANGED.
+    @event{EVT_COLOURPICKER_CURRENT_CHANGED(id, func)}
+           The user changed the currently selected colour in the dialog
+           associated with the control. This event is sent immediately when the
+           selection changes and you must also handle @c EVT_COLOUR_CANCELLED
```

Comment:
Shouldn't there be `EVT_COLOURPICKER_DIALOG_CANCELLED `instead of `EVT_COLOUR_CANCELLED`? I am also not sure how to decide whether to use `EVT_xxx` and not `wxEVT_xxx` when referring to events...

Reply:
Fixed the event type, thanks for noticing this one too!

As for `EVT_XXX` vs `wxEVT_XXX`, I don't really know myself, but as we're in an event table macros documentation block, I guess the wx-less version makes more sense here?